test/easy_localization_context_test.dart

```diff
@@ -38,8 +38,8 @@ class MyWidget extends StatelessWidget {
 }
 
 void main() async {
-  await EasyLocalization.ensureInitialized();
   SharedPreferences.setMockInitialValues({});
+  await EasyLocalization.ensureInitialized();
```

Comment:
This fixes the test failure due to the shared preferences null exception. Have to initialize shared preferences mock first.